src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2021_05_01/operations/_managed_clusters_operations.py

```diff
@@ -389,7 +389,7 @@ def get_access_profile(
     get_access_profile.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/accessProfiles/{roleName}/listCredential'}
 
     def list_cluster_admin_credentials(
-            self, resource_group_name, resource_name, custom_headers=None, raw=False, **operation_config):
+            self, resource_group_name, resource_name, server_fqdn=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
Why not upgrade the entire vendor SDK, but change only one file?

Reply:
I upgrade the entire sdk in this commit: https://github.com/Azure/azure-cli-extensions/pull/3453 
And then we have a hotfix for swagger to add this query param. That's why I update this one sdk file again based on latest swagger.

Reply:
OK, got it~